VAMobile/src/components/PhotoUpload.tsx

```diff
@@ -0,0 +1,171 @@
+import { Image, Pressable, PressableProps } from 'react-native'
+import { ImagePickerResponse, launchCamera, launchImageLibrary } from 'react-native-image-picker'
+import { useTranslation } from 'react-i18next'
+import React, { FC, ReactNode, useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import { NAMESPACE } from 'constants/namespaces'
+import { VAIcon } from './index'
+import { themeFn } from 'utils/theme'
+import { useDestructiveActionSheet, useShowActionSheet } from 'utils/hooks'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import Box, { BoxProps } from './Box'
+import TextView from './TextView'
+import theme from 'styles/themes/standardTheme'
+
+type PhotoUploadProps = {
+  /** width of the photo */
+  width: number
+  /** height of the photo */
+  height: number
+  /** imagePickerResponse with asset to style for component and fileSize */
+}
+
+type StyledImageProps = {
+  /** prop to set image width */
+  width: number
+  /** prop to set image height */
+  height: number
+  /** Hardcoded radius of 50 due to circular design plan */
+  borderRadius: number
+  /** Hardcoded radius of 2 due to design plan */
+  borderWidth: number
+  /** Hardcoded border color of white */
+  borderColor: string
+}
+
+const StyledImage = styled(Image)<StyledImageProps>`
+  width: ${themeFn<StyledImageProps>((_theme, props) => props.width)}px;
+  height: ${themeFn<StyledImageProps>((_theme, props) => props.height)}px;
+  border-radius: ${themeFn<StyledImageProps>((_theme, props) => props.borderRadius)}px;
+  border-width: ${themeFn<StyledImageProps>((_theme, props) => props.borderWidth)}px;
+  border-color: ${themeFn<StyledImageProps>((_theme, props) => props.borderColor)};
+`
+//TODO: Add this back to the VeteranStatusScreen when decided upon
+const PhotoUpload: FC<PhotoUploadProps> = ({ width, height }) => {
+  const { t } = useTranslation(NAMESPACE.COMMON)
+  const confirmAlert = useDestructiveActionSheet()
+  const photoUploadBorderRadius = 50
+  const photoUploadBorderWidth = 2
+  const showActionSheetWithOptions = useShowActionSheet()
+  const VETERAN_STATUS_PHOTO = '@store_veteran_status_photo'
+  const [uri, setUri] = useState('')
+  const options = [t('fileUpload.camera'), t('fileUpload.photoGallery'), t('cancel')]
+  const uploadBorderColor = theme.colors.border.photoUpload
+
+  const getPhotoFromStorage = async (): Promise<void> => {
+    const override = await AsyncStorage.getItem(VETERAN_STATUS_PHOTO)
+    if (override !== null) {
```

Comment:
Is `null` the only possible value when AsyncStorage doesn't find anything? If it's possible to be an empty string, or undefined, you could replace this with `if (!!override)`.

Reply:
Sure, seems fine.